app/api/services/domain.py

```diff
@@ -19,3 +20,6 @@ def get_by_name_or_id(self, name_or_id):
             domain = self.get(name_or_id).one_or_none()
 
         return domain
+
+    def get_all(self):
```

Comment:
there is already a `all(self)` in helpers.py -> Service class.  Although it is missing sorting, it might be enough for where your using it